Content.Server/Disposal/Unit/EntitySystems/DisposalUnitSystem.cs

```diff
@@ -471,6 +474,12 @@ public bool TryInsert(EntityUid unitId, EntityUid toInsertId, EntityUid userId,
             if (!Resolve(unitId, ref unit))
                 return false;
 
+            if (!HasComp<SharedHandsComponent>(userId) && toInsertId != userId) // Mobs like mouse can Jump inside even with no hands
+            {
+                _popupSystem.PopupEntity(Loc.GetString("disposal-unit-no-hands"), userId, Filter.Entities(userId));
```

Comment:
make this a caution popup (it's the optional 4th field for PopupEntity)

Reply:
Done